index.html

```diff
@@ -40,14 +40,11 @@
         <h1>Welcome to Malcolm’s CSD120 Website!</h1>
         
         
-        <picture>
-          <a href="Images/MalPic.jpg" alt ="Thumbnail">
-          <source media="(min-width: 300px)" srcset="Images/MalPicThumb300wide.jpg x2" alt="Thumbnail" />
-          <source media="(min-width: 150px)" srcset="Images/MalPicThumb150wide.jpg x1" alt = "Thumbnail"/>
-          <img src="Images/MalPicThumb300wide.jpg" alt=" Thumbnail" />
-         
-        </picture>
-
+        <a href="Images/MalPic.jpg">
+          <img src="Images/MalPicThumb300wide.jpg" 
+            sizes="(max-width: 500px) 150px, 300px"
+            srcset="Images/MalPicThumb150wide.jpg 150w, Images/MalPicThumb300wide.jpg 300w"
+            alt="A selfie of Malcolm" />
         </a>
```

Comment:
You're on the right track, but you can do this with just srcset and sizes in `<img>`, then wrap the image in a hyperlink that links to the full-size image